packages/block-library/src/site-logo/index.php

```diff
@@ -22,8 +22,8 @@ function render_block_core_site_logo( $attributes ) {
 	};
 
 	add_filter( 'wp_get_attachment_image_src', $adjust_width_height_filter );
-
 	$custom_logo = get_custom_logo();
+	remove_filter( 'wp_get_attachment_image_src', $adjust_width_height_filter );
```

Comment:
Does removing the filter here cause the width/height adjustments to not be applied? Since the filter is being removed before the logo is rendered, regardless of whether the logo is empty and returns early.